vyper/venom/analysis/dfg.py

```diff
@@ -22,6 +22,10 @@ def get_uses(self, op: IRVariable) -> list[IRInstruction]:
     def get_producing_instruction(self, op: IRVariable) -> Optional[IRInstruction]:
         return self._dfg_outputs.get(op)
 
+    def add_use(self, op: IRVariable, inst: IRInstruction):
+        uses = self._dfg_inputs.setdefault(op, [])
+        uses.append(inst)
```

Comment:
note: this could be out of order from recalculating dfg from scratch, maybe in the future we should use something like `set()` or `OrderedSet()` for uses